Adds info on stable and beta addon versions (#1978)
home-assistant/developers.home-assistant: docs/add-ons/presentation.md

For guidance on keeping a changelog, we recommend the [keep a changelog](http://keepachangelog.com) website. They have developed a standard used by many open source projects around the world.

## Offering stable and canary version

> You should consider to offer a stable and a "next" or "canary" branch. These can be provided using different branches. When adding the addon in Home Assistant the user can select the wanted branch from a given repository by appending its name following a hashtag.

Reviewer: ```suggestion
You may consider to offer a stable and a "next" or "canary" branch. These can be provided using different branches. When adding the add-on in Home Assistant, the user can select the wanted branch from a given repository by appending its name following a hashtag.
```

```text
https://github.com/home-assistant/hassio-addons-example#next
```

> You should add this information to your documentation. Also you should consider to have different [names for the repositories](/docs/add-ons/repository#repository-configuration) in every branch, e.g. "Super-Addon (Stable)" and "Super-Addon (Beta)".

Reviewer: ```suggestion
You should add this information to your documentation. Also, you should consider having different [names for the repositories](/docs/add-ons/repository#repository-configuration) in every branch, for example, "Super add-on (stable)" and "Super add-on (beta)".
```
